classes/Currency.php

```diff
@@ -787,6 +787,11 @@ public function refreshLocalizedCurrencyData(array $languages, LocaleRepository
             // CLDR currency gives data from CLDR reference, for the given language
             $cldrCurrency = $cldrLocale->getCurrency($this->iso_code);
 
+            if (empty($cldrCurrency)) {
+                // If the currency is not declared in the locale, we can't proceed
```

Comment:
"and if the locale is not found by CLDR, eg with custom iso code"
wdyt ?

Reply:
I updated the comment